manifests/s/Swift/Toolchain/5.7.2/Swift.Toolchain.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS0.CRLF.5-1-25267-1000.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Swift.Toolchain
+PackageVersion: 5.7.2
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: burn
+Scope: machine
+UpgradeBehavior: uninstallPrevious
+Commands:
+- swift
+FileExtensions:
+- swift
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+    MinimumVersion: 14.28.29913.0
+  - PackageIdentifier: Microsoft.WindowsSDK
+  - PackageIdentifier: Microsoft.VisualStudio.2019.BuildTools
+  - PackageIdentifier: Git.Git
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.swift.org/swift-5.7.2-release/windows10/swift-5.7.2-RELEASE/swift-5.7.2-RELEASE-windows10.exe
+  InstallerSha256: F523F8D364135738D9E7419F987A654A15159424434A13FC480079BCFCE78C75
```

```suggestion
  InstallerSha256: F523F8D364135738D9E7419F987A654A15159424434A13FC480079BCFCE78C75
  AppsAndFeaturesEntries:
    - DisplayVersion: 5.7.0
      DisplayName: Swift Developer Package for Windows x86_64
```
We can temporarily use AppsAndFeatureEntries and explicitly 1:1 match with Registry. Per my understanding, the consequence of this would be:
Users already on `5.7.1` (DisplayVersion in Registry of this release still being `5.7.0`) will receive an upgrade for 5.7.2 in WinGet but after installation WinGet will show they still have 5.7.0 installed and show 5.7.2 as available upgrade. (Upgrade loop)
It'll be confusing but it's the best we can do in this scenario.

---

Yeah, I can see how that is confusing.  But yes, that does seem like a good compromise for the moment.